aries-site/src/examples/templates/popover/PopoverInlineExample.js

```diff
@@ -0,0 +1,117 @@
+import React, { useState, useRef, useContext } from 'react';
+import { Popover } from 'aries-core';
+import {
+  Anchor,
+  Grid,
+  Card,
+  CardBody,
+  Button,
+  Box,
+  Paragraph,
+  Text,
+  Heading,
+  ResponsiveContext,
+} from 'grommet';
+import { CircleInformation } from 'grommet-icons';
+
+const device = [
+  {
+    title: 'Require Assignments',
+    devices: '10',
+  },
+  {
+    title: 'Require Subscriptions',
+    devices: '10',
+  },
+  {
+    title: 'Assigned & Subscribed',
+    devices: '0',
+  },
+  {
+    title: 'Total Devices',
+    devices: '10',
+  },
+];
+
+const columns = {
+  xsmall: ['auto'],
+  small: ['auto'],
+  medium: ['auto'],
+  large: ['auto', 'auto', 'auto'],
+  xlarge: ['auto', 'auto', 'auto', 'auto'],
+};
+
+// should we disable ?
+// eslint-disable-next-line react/prop-types
+const CardDevices = ({ title, devices }) => (
+  <Card>
+    <CardBody>
+      <Text>{title}</Text>
+      <Text>{devices}</Text>
+    </CardBody>
+  </Card>
+);
+
+export const PopoverInlineExample = () => {
+  const breakpoint = useContext(ResponsiveContext);
+  const [showPopover, setShowPopover] = useState(false);
+  const targetRef = useRef();
+
+  const handleButtonClick = () => {
+    setShowPopover(prev => !prev);
+  };
+
+  const handleClosePopover = () => {
+    setShowPopover(false);
+  };
+
+  return (
+    <Box gap="medium">
+      <Box
+        justify="between"
+        direction={!['xsmall', 'small'].includes(breakpoint) ? 'row' : 'column'}
+      >
+        <Box>
+          <Heading margin="none" level={2}>
+            Inventory
+          </Heading>
+          <Box direction="row">
+            <Text size="large">
+              View all available devices or add new devices.
+            </Text>
+            <Button
+              align="center"
+              justify="start"
+              icon={<CircleInformation size="small" />}
+              onClick={handleButtonClick}
+              ref={targetRef}
+              a11yTitle="Informational help"
+              aria-expanded={showPopover}
```

Comment:
Do we need `aria-controls` or `aria-owns` here as well?

Also does `aria-haspopup` make sense to include here?

Reply:
good question 

`aria-haspopup` tells users that there is a pop-up associated with the element, and it specifies the type of pop-up. This helps users know what to expect when they interact with the element.

`aria-controls` gives users more detailed information about which specific element will be affected when they interact with the element that has aria-haspopup.

so I think it is fair to add them, and recommend they be use. 
Use aria-controls in conjunction with other ARIA attributes like aria-expanded, aria-hidden, or aria-labelledby to provide a complete picture of the element's state and relationships.

I dont think we need the following:
`aria-owns` is used to define relationships in the DOM tree, but popovers are usually positioned outside the regular flow.